packages/genesys-spark-components/src/components/beta/gux-side-sheet/gux-side-sheet.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, h, Element, Prop } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxSideSheetSize } from './gux-side-sheet.types';
+
+@Component({
+  tag: 'gux-side-sheet-beta',
+  styleUrl: 'gux-side-sheet.scss',
+  shadow: true
+})
+export class GuxSideSheet {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  size: GuxSideSheetSize = 'small';
```

Comment:
If this is just the non modal version I think the sizing is wrong, there should only be small and medium